### ISSUE TITLE
Release? Code? Something?

### ISSUE THREAD

**Issue by turbo** [xoreaxeaxeax/..cantor.dust..#1]
Hi,

Is this ever going to be released?

Cheers.

**Comment by ashr** [comment]
Yeah man, we find your website and your presentations, but we want your code ! Really awesome concept.

**Comment by eXpl0it3r** [comment]
I was suggested to take a look at [Veles](https://codisec.com/veles/) just recently.

**Comment by cymerio** [comment]
There is also this website, but there are no downloads or signs of activity:
https://sites.google.com/site/xxcantorxdustxx/

Here's a link with a demo:
https://www.reddit.com/r/ReverseEngineering/comments/1izity/cantordust_a_binary_visualization_tool/

**Comment by SuperContraptionGuy** [comment]
Any word?  I am really interested in trying out this tool and getting to know some data types.

**Comment by 0x5c** [comment]
Same here, this tool would be very interesting, event if unfinished.

**Comment by AlexisTM** [comment]
This looks awesome. Hope it will be released.

**Comment by 0x5c** [comment]
@eXpl0it3r I tried veles, but it's lacking horribly.

**Comment by 0x5c** [comment]
It looks like we're going to have to write emails to @xoreaxeaxeax

**Comment by oaeide** [comment]
I have to say that I am somewhat disappointed that @xoreaxeaxeax isn't sharing this as he promised when he presented it. Might be a good reason for it but to my knowledge such a reason was never stated

**Comment by AlexisTM** [comment]
Just wait or develop it :o
